assets/js/components/KeyMetrics/ChipTabGroup/Chip.js

```diff
@@ -36,16 +40,34 @@ export default function Chip( {
 	onClick,
 	selectedCount = 0,
 } ) {
+	let icon = null;
+
+	switch ( slug ) {
+		case KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG:
+			icon = (
+				<CheckMark
+					width={ 12 }
+					height={ 12 }
+					className={ `googlesitekit-chip-tab-group__chip-item-svg googlesitekit-chip-tab-group__chip-item-svg__${ slug }` }
+				/>
+			);
+			break;
+		case KEY_METRICS_SUGGESTED_GROUP_SLUG:
+			icon = (
+				<StarFill
+					className={ `googlesitekit-chip-tab-group__chip-item-svg googlesitekit-chip-tab-group__chip-item-svg__${ slug }` }
+					width={ 12 }
+					height={ 12 }
+				/>
+			);
+			break;
+	}
```

Comment:
We can simplify this, since `width`, `height` and `className` will always be the same, instead of repeating it, we can use it directly as component props bellow, and here we can only switch between the icon component, for example:
```js
let Icon = null;

switch ( slug ) {
		case KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG:
			Icon = CheckMark;
			break;
		case KEY_METRICS_SUGGESTED_GROUP_SLUG:
			Icon = StarFill;
			break;
	}
```

Reply:
Thank you, @zutigrm , this has been refactored in my recent commits.